src/core/datastore/networkstore.js

```diff
@@ -24,6 +24,15 @@ export class NetworkStore {
      */
     this.collection = collection;
 
+    if (options.tag && !isString(options.tag)) {
+      throw new KinveyError('options.tag must be a string.');
```

Comment:
I don't think any nonempty string is a valid tag and we should tell the user so, to avoid unnecessary support queries.

Reply:
@georgiwe let's define what the validation will allow or not

Reply:
Please check my other comment on it. I imagine JS might be the restricting factor in this matter, cause of the numerous data storage providers and platforms. But some research is needed. Only one I feel confident would work everywhere at this point is [A-Za-z] :)